modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -521,6 +521,54 @@ public void testPutObjectDefinition() throws Exception {
 		_objectDefinitionLocalService.deleteObjectDefinition(
 			randomModifiableSystemObjectDefinition.getId());
 
+		// Publish a persisted draft custom object definition
+
+		ObjectDefinition randomPersistedDraftObjectDefinition =
```

Comment:
Reuse the variable `postObjectDefinition` here

Reply:
Done here https://github.com/liferay-objects/liferay-portal/pull/3054/commits/50f3d49f09e15eaa747edb3ad7a60b98e082acea